test/check-sosreport

```diff
@@ -25,7 +25,6 @@ import subprocess
 import unittest
 
 @unittest.skipIf("debian" in os.environ.get("TEST_OS", ""), "No sosreport on Debian.")
```

Comment:
You are right, I must have missed that somehow at the time.  I'll make a separate PR for this.